### PR TITLE
[couchbase-server] Get dates from docs.couchbase.com

### PR DESCRIPTION
**Pull request by marcwrobel**
Couchbase server was using the dockerhub auto method, which is not reliable. This new method is not perfect, but at least the dates are acceptable.

### CODE REVIEW

**Comment by marcwrobel** [comment]
Expected diff:

```diff
diff --git i/releases/couchbase-server.json w/releases/couchbase-server.json
index 30e23e0..cb4084a 100644
--- i/releases/couchbase-server.json
+++ w/releases/couchbase-server.json
@@ -1,54 +1,26 @@
 {
-  "7.2.0": "2023-06-16",
-  "7.1.4": "2023-06-16",
-  "7.0.5": "2023-06-16",
-  "6.6.6": "2023-06-16",
-  "7.1.3": "2023-03-02",
-  "7.0.4": "2022-12-09",
-  "6.6.5": "2022-12-09",
-  "7.1.2": "2022-10-26",
-  "7.1.1": "2022-10-05",
-  "7.0.3": "2022-10-05",
-  "7.1.0": "2022-06-07",
-  "6.5.2": "2022-04-30",
-  "6.0.5": "2022-04-30",
-  "6.6.4": "2022-01-07",
-  "7.0.2": "2021-10-16",
-  "6.6.3": "2021-10-16",
-  "7.0.1": "2021-10-01",
-  "7.0.0": "2021-08-31",
-  "6.6.2": "2021-07-26",
-  "6.6.1": "2021-07-26",
-  "6.6.0": "2021-07-26",
-  "6.5.1": "2021-07-26",
-  "6.5.0": "2021-07-26",
-  "6.0.4": "2021-07-26",
-  "6.0.3": "2021-07-26",
-  "6.0.2": "2021-07-26",
-  "6.0.1": "2021-07-26",
-  "6.0.0": "2019-01-23",
-  "4.6.5": "2018-10-19",
-  "5.5.2": "2018-10-19",
-  "5.5.1": "2018-09-06",
-  "5.5.0": "2018-07-26",
-  "5.1.1": "2018-07-17",
-  "4.6.4": "2018-04-28",
-  "5.1.0": "2018-04-28",
-  "5.0.1": "2018-01-25",
-  "4.6.3": "2017-11-18",
-  "3.1.6": "2017-08-10",
-  "4.6.2": "2017-08-10",
-  "4.6.1": "2017-05-15",
-  "4.6.0": "2017-03-01",
-  "4.5.1": "2017-01-21",
-  "4.5.0": "2016-09-26",
-  "3.1.5": "2016-07-22",
-  "4.1.1": "2016-06-01",
-  "2.5.2": "2016-05-06",
-  "3.0.2": "2016-05-06",
-  "3.0.3": "2016-05-06",
-  "3.1.0": "2016-05-06",
-  "3.1.3": "2016-05-06",
-  "4.0.0": "2016-05-06",
-  "4.1.0": "2016-05-06"
+  "7.2.0": "2023-06-01",
+  "7.1.4": "2023-03-15",
+  "6.6.6": "2023-01-15",
+  "7.0.5": "2022-12-15",
+  "7.1.3": "2022-11-15",
+  "7.1.2": "2022-10-15",
+  "7.1.1": "2022-07-15",
+  "7.0.4": "2022-06-15",
+  "7.1.0": "2022-05-15",
+  "6.6.5": "2022-01-15",
+  "7.0.3": "2021-12-15",
+  "6.6.4": "2021-12-15",
+  "7.0.2": "2021-10-15",
+  "7.0.1": "2021-09-15",
+  "6.6.3": "2021-08-15",
+  "7.0.0": "2021-07-15",
+  "6.6.2": "2021-04-15",
+  "6.5.2": "2021-02-15",
+  "6.6.1": "2020-12-15",
+  "6.6.0": "2020-08-15",
+  "6.5.1": "2020-04-15",
+  "6.5.0": "2020-01-15",
+  "6.0.1": "2019-02-15",
+  "6.0.0": "2018-10-31"
 }
\ No newline at end of file
```

File: src/couchbase-server.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+REGEX = r"^Release (?P<version>\d+\.\d+(\.\d+)?) \((?P<date>.+)\)$"
+URLS = [
+    "https://web.archive.org/web/20230519160357/https://docs.couchbase.com/server/",
+    "https://docs.couchbase.com/server",
+]
+FIXED_VERSIONS = {
+    "6.0.0": "2018-10-31",  # https://www.couchbase.com/blog/announcing-couchbase-6-0/
+    "6.0.1": "2019-02-15",  # https://web.archive.org/web/20190307191211/https://docs.couchbase.com/server/6.0/release-notes/relnotes.html
+    "7.2.0": "2023-06-01",  # https://www.couchbase.com/blog/couchbase-capella-spring-release-72/
+}
```
Comment (captn3m0) [inline src/couchbase-server.py:29]: Not for now, but thoughts about having a separate `patches` directory that holds corrections?

**Review by captn3m0 — APPROVED** [review]
LGTM. Dropping older versions seems okay here.

File: src/couchbase-server.py (hunk 1 above)
Comment (marcwrobel) [reply src/couchbase-server.py:29]: Not yet, I am thinking it makes more sense to keep it in the same script at the moment. But will think about it.

**Pull request merged**